jscomp/runtime/js_int.ml

```diff
@@ -64,39 +49,16 @@ The output will be rounded or padded with zeroes if necessary.
 @raise RangeError if digits is not in the range \[0, 20\] (inclusive)
 
 {[
-  (* prints "7.70e+1" *)
-  let _ = Js.log  (Js.Int.toExponentialWithPrecision 77 ~digits:2)
-
-  (* prints "5.68e+3" *)
-  let _ = Js.log  (Js.Int.toExponentialWithPrecision 5678 ~digits:2)
+  Js.Int.toExponential 77 = "7.7e+1"
+  Js.Int.toExponential ~digits:2 77 = "7.70e+1"
+  Js.Int.toExponential ~digits:2 5678 = "5.68e+3"
 ]}
 
 @see <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toExponential> MDN
 *)
 
-external toPrecision : int -> string = "toPrecision"
-[@@mel.send]
-(** Formats a [int] using some fairly arbitrary rules
-
-{b Returns} a [string] representing the given value in fixed-point (usually)
-
-[toPrecision] differs from [toFixed] in that the former will format the number
-with full precision, while the latter will not output any digits after the
-decimal point.
-
-@raise RangeError if digits is not in the range accepted by this function (what do you mean "vague"?)
-
-{[
-  (* prints "123456789" *)
-  let _ = Js.log (Js.Int.toPrecision 123456789)
-]}
-
-@see <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision> MDN
-*)
-(* equivalent to `toString` I think *)
-
-external toPrecisionWithPrecision : int -> digits:int -> string = "toPrecision"
-[@@mel.send]
+external toPrecision : ?digits:t -> string = "toPrecision"
+[@@mel.send.pipe: t]
 (** Formats an [int] using some fairly arbitrary rules
 
 {b digits} specifies how many digits should appear in total. The
```

Comment:
Maybe unrelated to the PR, but the part below is a bit unwarranted imo:

> for Node it's 21. Why? Who knows

Reply:
I fixed this. It was pretty easy to check the accepted range by passing it 0:


<img width="535" alt="image" src="https://github.com/melange-re/melange/assets/661909/87a8b471-c46e-436e-982f-a60a1fa69ec3">